Makefile

```diff
@@ -0,0 +1,7 @@
+###################################
+#   		Makefile - distar  				#
+#  			Copyright (C) 2018 			  #
+# Yan Régis-Gianas, Étienne Maris #
```

Comment:
There are some typos here.

Reply:
I've corrected this problem in 5cb96b5626504298f83f6d7c73f744919c8f3687. I don't know if I can change in the current commit.